src/shared/components/NotFound.tsx

```diff
@@ -1,7 +1,7 @@
-import React, {SFC} from 'react'
+import React, {FC} from 'react'
 
-const NotFound: SFC = () => (
-  <div className="container-fluid">
+const NotFound: FC = () => (
+  <div className="container-fluid" data-testid="404">
```

Comment:
small suggestion to keep the naming consistent: the component and the concept is called `NotFound`, not 404. let's not mix terms / levels of abstraction.

```suggestion
  <div className="container-fluid" data-testid="component-not-found">
```